src/game-coordinator/services/server-configurator.service.ts

```diff
@@ -42,8 +41,24 @@ export class ServerConfiguratorService {
     private gamesService: GamesService,
     private gameServersService: GameServersService,
     private gameConfigsService: GameConfigsService,
+    private events: Events,
   ) {}
 
+  onModuleInit() {
+    // when a gameserver is assigned to a game, we can configure the gameserver
+    this.events.gameChanges
+      .pipe(
+        filter(({ newGame }) => !!newGame.gameServer),
+        filter(
+          ({ oldGame, newGame }) =>
+            JSON.stringify(oldGame.gameServer) !==
+            JSON.stringify(newGame.gameServer),
```

Comment:
poverty deep equals, I'd just use lodash or whatever equivelant there is

Reply:
this could actually throw false positives in the wild, I wouldn't actually rely on this moving forward

Reply:
+1
I'll have that fixed